vortex-datetime-dtype/src/unit.rs

```diff
@@ -26,6 +27,18 @@ pub enum TimeUnit {
     D,
 }
 
+impl TimeUnit {
+    pub fn to_jiff_span(&self, v: i64) -> Span {
+        match self {
+            TimeUnit::Ns => Span::new().nanoseconds(v),
+            TimeUnit::Us => Span::new().microseconds(v),
+            TimeUnit::Ms => Span::new().milliseconds(v),
+            TimeUnit::S => Span::new().seconds(v),
+            TimeUnit::D => Span::new().days(v),
```

Comment:
Note that these routines will panic if `v` is out of bounds.

Reply:
Thanks! I ought to have looked more carefully at the docs.